File: bitget-golang-sdk-api/pkg/model/spot/order/openordersreq.go
package order

/**
 * @Author: bitget-sdk-team
 * @Date: 2022-09-30 10:46
 * @DES: Obtain orders that have not been closed or partially closed but not cancelled request
 */
type OpenOrdersReq struct {

	/**
	 * Currency pair
	 */
	Symbol string `json:"symbol"`
}

File: bitget-php-sdk-api/src/model/spot/order/OpenOrdersReq.php
<?php


namespace bitget\model\spot\order;


class OpenOrdersReq
{
    /**
     * Currency pair
     */
    var string $symbol;

    /**
     * @return string
     */
    public function getSymbol(): string
    {
        return $this->symbol;
    }

    /**
     * @param string $symbol
     */
    public function setSymbol(string $symbol): void
    {
        $this->symbol = $symbol;
    }


}